sematic/ui/package.json

```diff
@@ -54,7 +54,11 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "analyze": "source-map-explorer 'build/static/js/*.js'",
-    "lint": "eslint --config node_modules/eslint-config-react-app/index.js --ext .ts,.tsx --max-warnings=0 ./src"
+    "lint:app": "eslint --config node_modules/eslint-config-react-app/index.js --ext .ts,.tsx --max-warnings=0 ./src",
```

Comment:
Please document these steps in the developer docs.

Reply:
Sounds good. Since this PR has been approved, I will do both (pre-commit + developer docs) in a separate PR.